reactor-netty-http/src/main/java/reactor/netty/http/server/HttpServerOperations.java

```diff
@@ -295,7 +295,8 @@ public HttpServerResponse headers(HttpHeaders headers) {
 	@Override
 	public boolean isFormUrlencoded() {
 		String contentType = requestHeaders().get(HttpHeaderNames.CONTENT_TYPE);
-		return HttpHeaderValues.APPLICATION_X_WWW_FORM_URLENCODED.contentEqualsIgnoreCase(contentType);
+		return contentType != null &&
+				contentType.startsWith(HttpHeaderValues.APPLICATION_X_WWW_FORM_URLENCODED.toString());
```

Comment:
I would suggest avoiding `startsWith` and instead maybe introducing some sort of headers sanitization or comparison util which removes/ignores directives... as you said offband, the `;` can be separated by whitespace so it is not a simple matter of checking the n-th character, unfortunately :(

Reply:
good catch, let me fix that

Reply:
please check now